contract/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -45,8 +45,8 @@ public NeoURI(Neow3j neow3j) {
      * @throws IllegalFormatException if the provided URI has an invalid format.
      */
     public static NeoURI fromURI(String uriString) throws IllegalFormatException {
-        String[] base_and_query = uriString.split("\\?");
-        String[] beginTx = base_and_query[0].split(":");
+        String[] baseAndQuery = uriString.split("\\?");
```

Comment:
What if the `uriString` is null? NullPointerException? 😄 

Add a check for this, please. Also, a small unit test to make sure.

Reply:
Yes, I change that in the next commit. I now throw an IllegalArgumentException if uriString is null.